test_package/.analysis_options

```diff
@@ -0,0 +1 @@
+{ analyzer: { exclude: ['**'] } }
```

Comment:
I think of this file as just a 1-line marker file - don't look in this directory.

Hopefully the AS issue I filed will be addressed - the parent pubspec can exclude this directory, and this file can just be deleted.